lldb/test/API/tools/lldb-dap/progress/Progress_emitter.py

```diff
@@ -37,7 +37,11 @@ def create_options(cls):
         )
 
         parser.add_option(
-            "--total", dest="total", help="Total to count up.", type="int"
+            "--total",
+            dest="total",
+            help="Total to count up, use None to identify as indeterminate",
```

Comment:
"Total items in this progress object. When this option is not specified, this will be an indeterminate progress."